src/common/DOMWorld.ts

```diff
@@ -292,12 +292,12 @@ export class DOMWorld {
     }
 
     if (path !== null) {
-      if (!isNode) {
+      const fs = Config.fs;
+      if (!fs) {
```

Comment:
The problem with this change is that one of the things we have planned is that we will bundle a web version of Puppeteer that uses a tool like Rollup to minify and eliminate dead code.

Our plan was to have a bundle step where Rollup can eliminate any `if (isNode)` code branches as we'll tell it that `isNode` is false.

I wonder if we should change this to be `if (!Environment.isNode)` because then we can eliminate these branches at bundling time for our web version.

Reply:
I agree with you about using `if(!Environment.xxx)` to check for dead code, but I don't think `isNode` is the good choice, because we can config the `fs` module for the web if needed(that's more flexible), so we can't say it's always dead code,  it is determined by `fs` itself not `isNode`.
If `if (!Environment.fs)` works in bundle tools, I think it's better.